### PR TITLE
Updated roles: init_dbserver, and setup_pgd

### PR DESCRIPTION
**Pull request by dougortiz**
Updated roles: 
- init_dbserver
- setup_pgd

All tasks related to: 
- PGD
- PGD CAMO
- PGD GROUP COMMIT SCOPES

### CODE REVIEW

File: roles/init_dbserver/tasks/setup_tde.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@
   become: true
   when:
     - edb_master_key|length > 0
-    - pg_data not in edb_master_key_location
+    - edb_master_key_location not in pg_data
```
Comment (vibhorkumar123) [inline roles/init_dbserver/tasks/setup_tde.yml:12]: Why do you have this change? I doubt it's intentional. Please check your branch and rebase accordingly.

Comment (vibhorkumar123) [reply roles/init_dbserver/tasks/setup_tde.yml:12]: This doesn't look correct to me.

File: roles/setup_pgd/tasks/pg_hba_config.yml [hunk 2]
```diff
@@ -57,6 +56,52 @@
...
+          "databases": pg_database
+          }
+        ] }}
+  when: pg_ssl
+  loop: "{{ pgd_cluster_nodes }}"
+  loop_control:
+    loop_var: node
+  run_once: true
+  no_log: "{{ disable_logging }}"
+
```
Comment (vibhorkumar123) [inline roles/setup_pgd/tasks/pg_hba_config.yml:104]: IIRC, we verified that this is not needed for configuration. Any reason for this to be here

File: roles/setup_pgd/tasks/setup_tde.yml [hunk 3]
```diff
@@ -9,7 +9,7 @@
   become: true
   when:
     - edb_master_key|length > 0
-    - pg_data not in edb_master_key_location
+    - edb_master_key_location not in pg_data
```
Comment (vibhorkumar123) [inline roles/setup_pgd/tasks/setup_tde.yml:12]: This doesn't seem correct to me. Please correct it.

File: roles/setup_pgd/tasks/verify_tde_vars.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
-- name: Verify edb_key_wrap_cmd and edb_key_unwrap_cmd
+- name: Verify edb_key_wrap_cmd is defined or not
   ansible.builtin.fail:
-    msg: "edb_key_wrap_cmd and edb_key_unwrap_cmd are must for initializing a cluster with TDE"
+    msg: "edb_key_wrap_cmd and edb_key_unwrap_cmd are must for initializing a cluster"
```
Comment (vibhorkumar123) [inline roles/setup_pgd/tasks/verify_tde_vars.yml:4]: Change is not needed

**Review by vibhorkumar123 — CHANGES_REQUESTED** [review]

**Review by hannahms — CHANGES_REQUESTED** [review]
Where are the commit scope files?

File: roles/setup_pgd/README.md [hunk 5]
```diff
@@ -63,6 +63,44 @@ The rest of the variables can be configured and are available in the:
...
 
+### `PGD Commit Scopes Configuration`
+
+See examples for PGD Commit Scopes available at: [EDB PGD v5](https://www.enterprisedb.com/docs/pgd/5/durability/commit-scopes/).
+
+The code below is part of the [roles/setup_pgd/defaults/main.yml](./defaults/main.yml), and 
+example for configuring two PGD commit scopes is listed below.
+
+The configuration requirements for PGD through the configuration setting variables are:
+  1. Only one scope can be configured as default at a time. The variable to configure is: `default_group_cs`
```
Comment (hannahms) [inline roles/setup_pgd/README.md:74]: Remove this line

File: roles/setup_pgd/tasks/pgd_group_commitscope_configuration.yml [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+- name: Create PGD Cluster group commit scope origin node group
+  ansible.builtin.include_role:
+    name: manage_dbserver
+    tasks_from: execute_sql_scripts
+  vars:
+    pg_query:
+      - query: "SELECT bdr.create_node_group(node_group_name := '{{ item.cs_origin_node_group }}',
+                                      parent_group_name := '{{ item.parent_group }}',
+                                      join_node_group := false
```
Comment (hannahms) [inline roles/setup_pgd/tasks/pgd_group_commitscope_configuration.yml:37]: `join_node_group := true`

File: roles/init_dbserver/tasks/verify_tde_vars.yml [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ---
-- name: Verify edb_key_wrap_cmd and edb_key_unwrap_cmd
+- name: Verify edb_key_wrap_cmd and edb_key_unwrap_cmd are defined
   ansible.builtin.fail:
-    msg: "edb_key_wrap_cmd and edb_key_unwrap_cmd are must for initializing a cluster with TDE"
+    msg: "edb_key_wrap_cmd and edb_key_unwrap_cmd are must for initializing a cluster"
```
Comment (hannahms) [inline roles/init_dbserver/tasks/verify_tde_vars.yml:4]: should include something about `with TDE`, these are not required for initializing a cluster, but are for initializing a cluster with TDE.

File: roles/setup_pgd/README.md [hunk 8]
```diff
@@ -93,6 +130,8 @@ This role does not have any dependencies, but package repositories should have b
 configured beforehand with the `setup_repo` role. At least one lead primary must exist
 and a database cluster must be initialized on that node. 
 
+
+
```
Comment (hannahms) [inline roles/setup_pgd/README.md:134]: remove blank lines

File: roles/setup_pgd/tasks/verify_tde_vars.yml [hunk 9]
```diff
@@ -1,7 +1,7 @@
 ---
-- name: Verify edb_key_wrap_cmd and edb_key_unwrap_cmd
+- name: Verify edb_key_wrap_cmd and edb_key_unwrap_cmd are defined
   ansible.builtin.fail:
-    msg: "edb_key_wrap_cmd and edb_key_unwrap_cmd are must for initializing a cluster with TDE"
+    msg: "edb_key_wrap_cmd and edb_key_unwrap_cmd are must for initializing a cluster"
```
Comment (hannahms) [inline roles/setup_pgd/tasks/verify_tde_vars.yml:4]: Same comment as in init_dbserver

**Review by hannahms — CHANGES_REQUESTED** [review]
Update verify_tde_vars and README

File: roles/setup_pgd/tasks/verify_tde_vars.yml (hunk 9 above)
Comment (dougortiz) [reply roles/setup_pgd/tasks/verify_tde_vars.yml]: Updated

**Review by hannahms — APPROVED** [review]
LGTM

**Review by vibhorkumar123 — COMMENTED** [review]
LGTM!

**Pull request merged**